RickAndMorty/Views/RMDetailEpisode/DetailEpisodeViewModel.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  DetailEpisodeViewModel.swift
+//  RickAndMorty
+//
+//  Created by Juan Camilo Fonseca Gomez on 3/09/23.
+//
+
+import Foundation
+import Observation
```

Comment:
Double Check the Observation API, You no longer need to add ObservableObject conformation, Now the approach is to add a Property Wrapper

Reply:
I removed the import and conformation